src/server.js

```diff
@@ -54,17 +52,20 @@ app
   .then(() => {
     const server = express();
 
-    const { SESSION_SECRET, SESSION_MAX_AGE_MS } = process.env;
+    const { SESSION_MAX_AGE_MS } = process.env;
     const maxAge = SESSION_MAX_AGE_MS ? Number(SESSION_MAX_AGE_MS) : 24 * 60 * 60 * 1000; // 24 hours
 
     // We use a client-side cookie session instead of a server session so that there are no
     // issues when load balancing without sticky sessions.
     // https://www.npmjs.com/package/cookie-session
+    // The cookie is not `signed` because it's value is to be changed when access_token is
+    // refreshed in the browser.
     server.use(cookieSession({
       // https://www.npmjs.com/package/cookie-session#options
-      keys: [SESSION_SECRET],
       maxAge,
-      name: "storefront-session"
+      name: "storefront-session",
+      httpOnly: false,
+      signed: false
```

Comment:
Removing `httpOnly` does worry me a little, but there may be no other way. Let's document this in plain English and have a couple people review the approach in the docs before merging this. For now, can you just add an `auth.md` file in `docs` in this repo?

Docs should explain
- how cookies are used for auth
- what type of cookies and why
- what is happening to set the cookie during login
- how do the Apollo requests use the cookie/token
- how do we clear the cookie/state at logout
- what happens when token is expired while querying from server code
- what happens when token is expired while querying from client code

Reply:
Good point. I'll write out docs for it